doc/source/conf.py

```diff
@@ -389,6 +391,22 @@
 # -----------------------------------------------------------------------------
 
 nb_execution_mode = "auto"
+# Ignore notebooks generated by jupyterlite-sphinx for interactive examples.
+nb_execution_excludepatterns = ["_contents/*.ipynb"]
+
+#------------------------------------------------------------------------------
+# Interactive examples with jupyterlite-sphinx
+#------------------------------------------------------------------------------
+global_enable_try_examples = True
+try_examples_global_button_text = "Try it in your browser!"
+try_examples_global_warning_text = (
+    "SciPy's interactive examples with Jupyterlite are experimental and may"
+    " not always work as expected. Execution of cells containing imports may"
+    " result in large downloads (up to 60MB of content for the first import"
```

Comment:
Maybe adding a note that those data will then be cached by the browser, so the following openings of notebooks will not download those data again.

Reply:
Huh. I wonder if it's something to do with the tracking settings I have in firefox, but the caching doesn't work for me. Every time I import scipy in a different examples notebook, it downloads scipy, numpy, and openblas again.

Reply:
Yeah I would keep this as is. These are "details" and until we remove the experimental note we should not give more guarantee. At most i think we could add a link to the underlying tech.

Reply:
I've been playing around with the network tab in the Firefox web console. I've turned off all tracking protection in firefox, and the caching still doesn't work for me. Every time I execute an import from scipy in a different notebook, scipy, numpy, and openblas get downloaded again. Also, total download size from hitting the button to open the notebook and importing SciPy is actually more like 130MB. I was just adding up the size of the numpy, scipy, and openblas wheels that get downloaded before.

Reply:
Something weird was going on with the 130MB, I haven't witnessed that again